installation-bundle/src/Controller/InstallationController.php

```diff
@@ -361,15 +346,48 @@ private function setUpDatabaseConnection(): Response
             ));
         }
 
-        $dumper = new ParameterDumper($this->getContainerParameter('kernel.project_dir'));
-        $dumper->setParameters($parameters);
-        $dumper->dump();
-
-        $this->purgeSymfonyCache();
+        $this->storeDatabaseUrl($parameters);
 
         return $this->getRedirectResponse();
     }
 
+    /**
+     * Stores the database URL in the .env file.
+     */
+    private function storeDatabaseUrl(array $parameters): void
```

Comment:
@aschempp Does it make sense to encapsulate this in a `DotEnvDumper` class which can be re-used by the manager bundle?